@amifun/getArgs.m

```diff
@@ -78,11 +78,11 @@
         case 'y'
             this.argstr = '(realtype t, int it, realtype *y, N_Vector x, void *user_data)';
         case 'z'
-            this.argstr = '(realtype t, int ie, int *nroots, realtype *z, N_Vector x, void *user_data)';
+            this.argstr = '(realtype t, int ie, realtype *z, N_Vector x, void *user_data, void *temp_data)';
```

Comment:
for some functions UserData needs to be passed as void as the pointer is handled by CVODES/IDAS. For all other it's true that the casting is not necessary and I will fix this in the future.